server/redirect-url-helper.js

```diff
@@ -15,8 +15,8 @@ function processRedirects(req, res, next, sourceUrlArray, urls) {
   const search = query.search || "";
 
   sourceUrlArray.some((sourceUrl) => {
-    try {
-      if (urls[sourceUrl]) {
+    if (urls[sourceUrl]) {
```

Comment:
check if there is anycase if urls is not an array as its moved out of try block

Reply:
added a check